apps/web/src/design-system/typography/title/Title.tsx

```diff
@@ -0,0 +1,15 @@
+import { TitleOrder, Title as MantineTitle } from '@mantine/core';
+
+interface ITitleProps extends JSX.ElementChildrenAttribute {
+  order?: TitleOrder;
```

Comment:
@ainouzgali what do you think about mantine `order` prop? I find it a bit confusing, maybe something like `size` is better here?

Reply:
Yes, I've been thinking the same thing. 
Should we call it size with values 'md' and 'lg' to be consistent with the other components?

Reply:
I think it's better use 1,2,3,4,5 (now only 1,2) for a better semantic and mental model resemblance with h1, h2, h3 and etc...